cmd/osquery-perf/agent.go

```diff
@@ -2401,16 +2411,27 @@ func (a *agent) submitLogs(results []resultLog) error {
 		return fmt.Errorf("/version check failed: %w", err)
 	}
 
-	var resultLogs []byte
```

Comment:
This was an attempt to reduce memory/CPU footprint on a previous load test where we had to simulate Fleet server being down and a lot of logs buffered in osquery perf. Is this causing any issues?

Reply:
Oh no it wasn't, I just thought it looked odd. If it was there for a reason I'll put it back

Reply:
OK, yeah, it does look odd :), let's just put it back just to decrease risk of breaking loadtest (it takes time to setup ~100k-hosts load tests :))